examples/prom/nonlinear_elasticity_global_rom.cpp

```diff
@@ -1611,50 +1796,84 @@ void RomOperator::Mult_Hyperreduced(const Vector& vx, Vector& dvx_dt) const
     CAROM::Vector dv_dt_librom(dv_dt.GetData(), rvdim, false, false);
     CAROM::Vector dx_dt_librom(dx_dt.GetData(), rxdim, false, false);
 
-    // Lift the x-, and v-vectors
-    // I.e. perform v = v0 + V_v v^, where v^ is the input
-    V_v_sp->mult(v_librom, *z_v_librom);
-    V_x_sp->mult(x_librom, *z_x_librom);
+    if (eqp)
+    {   // Lift v-vector and save
```

Comment:
Perhaps comment here that it is lifting to FOM space, and that this could be further optimized by using a reduced mesh consisting of only the elements containing EQP points.

Reply:
Maybe that should go into the ComputeReducedEQP function instead? I don't know how this one here could be optimized. This is the dx/dxt = v part

Reply:
Optimizing by avoiding FOM data structures and only constructing everything on a "sample mesh" of elements with EQP points is difficult to implement. I am doing it for Laghos, and it is a large, involved code change. It should not be done in this PR, but at least a comment about the possibility would be helpful.